### PR TITLE
Map Redirects in mapSeamles [Tpwdcee 4370]

### PR DESCRIPTION
**Pull request by tomazpu**

### CODE REVIEW

File: src/Transaction/Transaction.php [hunk 1]
```diff
@@ -569,6 +569,14 @@ public function setRedirect(Redirect $redirect)
         return $this;
     }
 
+    /**
+     * @return Redirect
```
Comment (serucee) [inline src/Transaction/Transaction.php]: Add since tag

File: src/Entity/Redirect.php [hunk 2]
```diff
@@ -49,6 +49,11 @@ class Redirect
      */
     private $cancelUrl;
 
+    /**
+     * @var string
+     */
```
Comment (serucee) [inline src/Entity/Redirect.php]: Add since tag

File: src/Mapper/RequestMapper.php [hunk 3]
```diff
@@ -159,6 +159,19 @@ public function mapSeamlessRequest(Transaction $transaction, $requestData)
             $requestData = array_merge($requestData, $periodic->mappedSeamlessProperties());
         }
 
+        $redirects = $transaction->getRedirect();
+        if (null !== $redirects) {
+            if (null !== $redirects->getSuccessUrl()) {
+                $requestData['success-redirect-url'] = $redirects->getSuccessUrl();
```
[the remark this answers is not in the document]
Comment (tomazpu) [reply src/Mapper/RequestMapper.php]: Thank you

File: src/Mapper/RequestMapper.php [hunk 4]
```diff
@@ -159,6 +159,19 @@ public function mapSeamlessRequest(Transaction $transaction, $requestData)
...
+            }
```
[the remark this answers is not in the document]
Comment (tomazpu) [reply src/Mapper/RequestMapper.php]: Good idea

**Pull request merged**